crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -1006,6 +1007,27 @@ pub(crate) fn expression(expr: &Expr, checker: &mut Checker) {
                     pyupgrade::rules::unicode_kind_prefix(checker, string_literal);
                 }
             }
+            for element in value.elements() {
+                let Some(literal_element @ ast::FStringLiteralElement { value, range }) =
+                    element.as_literal()
+                else {
+                    continue;
+                };
+                if checker.enabled(Rule::HardcodedBindAllInterfaces) {
+                    flake8_bandit::rules::hardcoded_bind_all_interfaces(checker, value, *range);
```

Comment:
It always triggered for such cases, we're just maintaining the behavior. The reason is that it could be useful for cases like `f"0.0.0.0{port}/path"`: https://play.ruff.rs/7b6e80ea-5858-4983-877c-1669a5cd538d